features/admin.rules.v1/data.ts

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ConditionExpressionsMetaDataInterface, RuleExecutionMetaDataInterface } from "./models/meta";
+import { RuleExecuteCollectionInterface, RuleInterface } from "./models/rules";
+
+export const sampleRuleExecuteInstance: RuleInterface = {
```

Comment:
Hope this is sample data for local development and we can remove them from the PR

Reply:
This will be use for testing in the future hence I have moved it to `__tests__/__mocks__`